Pod/Classes/VOKManagedObjectMap.h

```diff
@@ -32,6 +34,16 @@
 #   define VOK_MAP_FOREIGN_TO_LOCAL(inputKeyPath, coreDataSelectorSymbol) [VOKManagedObjectMap mapWithForeignKeyPath:inputKeyPath coreDataKey:VOK_CDSELECTOR(coreDataSelectorSymbol)]
 #endif
 
+/**
+ *  Creates a map for a string to a number using a default NSNumberFormatter.
+ *
+ *  @param inputKeyPath             The foreign key to match with the local key.
+ *  @param coreDataSelectorSymbol   The local selector symbol
+ */
+#ifndef VOK_MAP_FOREIGN_STRING_TO_LOCAL_NUMBER
+#   define VOK_MAP_FOREIGN_STRING_TO_LOCAL_NUMBER(inputKeyPath, coreDataSelectorSymbol) [VOKManagedObjectMap mapWithForeignKeyPath:inputKeyPath coreDataKey:VOK_CDSELECTOR(coreDataSelectorSymbol) numberFormatter:[NSNumberFormatter new]]
+#endif
```

Comment:
I think I'm confused here - wouldn't this mean that the same number formatter would be used for every single object? Or are you saying this would be a per-object dispatch_once?